test/extended/workspacetype/workspace.go

```diff
@@ -16,49 +18,90 @@ var _ = g.Describe("[sig-workspace]", func() {
 		k = exutil.NewCLIWithWorkSpace("kcp-workspace")
 	)
 
-	g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should works well", func() {
-		g.By("# Create a test workspace under user home workspace should become ready to use")
+	g.It("Author:pewang-Medium-[Smoke] Multi levels workspaces lifecycle should work", func() {
+		g.By("# Create a test workspace as parent workspace")
 		k.SetupWorkSpace()
-		myWorkSpace := k.WorkSpace()
+		parentWorkSpace := k.WorkSpace()
 
-		g.By("# Create a child level workspace should become ready to use")
-		k.SetupWorkSpaceWithSpecificPath(myWorkSpace.ServerURL)
-		mySubWorkSpace := k.WorkSpace()
+		g.By("# Create five child workspaces under the parent workspace")
+		childWorkSpaces := []exutil.WorkSpace{}
+		for i := 1; i <= 5; i++ {
+			g.By(fmt.Sprintf("# Creating child workspace no.%v", i))
+			k.SetupWorkSpaceWithSpecificPath(parentWorkSpace.ServerURL)
+			childWorkSpace := k.WorkSpace()
+			childWorkSpaces = append(childWorkSpaces, childWorkSpace)
+		}
 
-		g.By("# From the home workspace could get the test workspace but couldn't get its child level workspace")
-		output, err := k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+myWorkSpace.ParentServerURL, "workspace").Output()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(output).Should(o.And(
-			o.ContainSubstring(myWorkSpace.Name),
-			o.ContainSubstring(myWorkSpace.ServerURL),
-		))
-		o.Expect(output).ShouldNot(o.ContainSubstring(mySubWorkSpace.Name))
-
-		g.By("# From the test workspace could get its child workspace")
-		output, err = k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("get").Args("--server="+myWorkSpace.ServerURL, "workspace").Output()
-		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(output).Should(o.And(
-			o.ContainSubstring(mySubWorkSpace.Name),
-			o.ContainSubstring(mySubWorkSpace.ServerURL),
-		))
-
-		g.By("# Delete the child workspace and the test workspace should be successful")
-		// Delete the child workspace and check it deleted successfully
-		output, err = k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("delete").Args("--server="+myWorkSpace.ServerURL, "workspace", mySubWorkSpace.Name).Output()
+		g.By("# From the home workspace could get parent workspace but couldn't get child workspaces")
+		homeSubWorkspaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ParentServerURL)
```

Comment:
For `orkspace` in your variable names, sometimes you use lower case `s`, sometimes you use uppercase S (e.g. `childWorkSpaces`). This is inconsistent. Could you check all and it is better to make them consistent?